### PR TITLE
arch: install: only use t2linux/archiso-t2 ISO

### PR DESCRIPTION
**Pull request by Redecorating**
Before there were a bunch of notes for when using the the
t2linux/archiso-t2 ISO. Made these the default instructions, and removed
instructions for the older ISO. This means users won't have to worry
about getting wired internet.

This includes changing the repo added to my one, but once we get updated
kernels and so forth on the dl.t2linux.org one, that should be changed
back.

Additionally, the archinstall script is very outdated, I don't know
if/when I'll update it, so I've removed the link to it.

### CODE REVIEW

**Review by NoaHimesaka1873 — CHANGES_REQUESTED** [review]
read the comments please

File: docs/distributions/arch/installation.md
Comment (Redecorating) [reply docs/distributions/arch/installation.md:38]: I don't think systemd-boot actually needs efibootmgr: https://github.com/systemd/systemd/search?q=efibootmgr (I think it uses sysfs to set nvram variables).

Grub has it as an optional dependency  "efibootmgr: For grub-install EFI support", so it does use efibootmgr

Comment (NoaHimesaka1873) [reply docs/distributions/arch/installation.md:38]: yeah, seems like systemd-boot doesnt use efibootmgr to write efivars.

**Review by NoaHimesaka1873 — APPROVED** [review]
Seems good.

File: docs/distributions/arch/installation.md
Comment (networkException) [inline docs/distributions/arch/installation.md:33]: Maybe it should be made more clear that you shouldn't format the APFS partitions

**Review by networkException — APPROVED** [review]
LGTM

**Comment by AdityaGarg8** [comment]
This PR is open for quite sometime.

**Comment by EmreTech** [comment]
Why is that? Networkexception approved it, so merge it.

**Comment by networkException** [comment]
Have all questions in the review conversations been answered?

File: docs/distributions/arch/installation.md
Comment (NoaHimesaka1873) [reply docs/distributions/arch/installation.md:33]: Depends. Should say not to delete nvme0n1p2 when intending to dual/triple boot

**Comment by Redecorating** [comment]
I don't really want to change the pacman repo without a review from @aunali1

If we can get new packages on dl.t2linux.org then there'd be no need to change it which would be better

**Comment by sharpenedblade** [comment]
I dont think @aunali1 is going to update the kernel soon, at least from what I can see in the repo. The wiki for arch makes it seem like the situation is far worse than it actually is, because of an outdated kernel. A lot of users use the Redecorating kernel anyways.

**Comment by AdityaGarg8** [comment]
Have all conversions been resolved here?

**Comment by NoaHimesaka1873** [comment]
> Have all conversions been resolved here?

Everything's resolved except waiting for aunali1's review. Don't think he's gonna review this anytime soon (He didn't for FIVE MONTHS) so merging would be fine.

**Pull request merged**
